lib/helpers/project-init.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(options) {
   var presets = initPresets[target]();
   var projectName = (target === 'addon') ? 'my-addon' : 'my-app';
   var cliOptions = {
-    package: options.package || path.resolve(process.cwd(), '..', '..', 'tests', 'fixtures', type, 'package'),
+    package: options.package || path.resolve(process.cwd(), '..', '..', 'node-tests', 'fixtures', type, 'package'),
```

Comment:
it should be noted that the `tmp` we're looking for is setup by the `beforeEach` hook in `setup.js`: 
https://github.com/ember-cli/ember-cli-blueprint-test-helpers/blob/master/lib/helpers/setup.js#L59